cmd/bb_worker/main.go

```diff
@@ -13,8 +13,8 @@ import (
 
 	re_blobstore "github.com/buildbarn/bb-remote-execution/pkg/blobstore"
 	"github.com/buildbarn/bb-remote-execution/pkg/builder"
-	"github.com/buildbarn/bb-remote-execution/pkg/configuration/bb_worker"
 	cas_re "github.com/buildbarn/bb-remote-execution/pkg/cas"
+	configuration "github.com/buildbarn/bb-remote-execution/pkg/configuration/bb_worker"
```

Comment:
What is the rationale of renaming this import?

Reply:
My VS Code auto-formatter did that. Have reverted it.